include/fpcsc/perf_util/feature_cmd.h

```diff
@@ -2,19 +2,42 @@
 #define FPCSC_PERFUTIL_FEATURECMD_H
 
 #include <string>
+#include <stdlib.h>
+#include <exception>
 
-namespace fpcsc {
-
-inline bool isFeatureEnabled(int argc, char* argv[], std::string FeatureName) {
-  for (int CurrentArg = 1; CurrentArg < argc; ++CurrentArg) {
-    if (argv[CurrentArg] == FeatureName) {
-      return true;
+namespace fpcsc
+{
+  inline bool isFeatureEnabled(int argc, char *argv[], std::string FeatureName)
+  {
```

Comment:
pls fix the formatting, from what I see in the diff, you just reformatted the file here